pkg/api/v1alpha1/cloudstackdatacenterconfig.go

```diff
@@ -18,13 +18,9 @@ func NewCloudStackDatacenterConfigGenerate(clusterName string) *CloudStackDatace
 		},
 		Spec: CloudStackDatacenterConfigSpec{
 			Domain: "domain1",
-			Zone: CloudStackResourceRef{
-				Value: "zone1",
-				Type:  "name",
-			},
-			Network: CloudStackResourceRef{
-				Value: "net1",
-				Type:  "name",
+			Zones: []Zone{
+				{ZoneRef: CloudStackResourceRef{Type: "name", Value: "zone1"}, Network: CloudStackResourceRef{Type: "name", Value: "GuestNet1"}},
+				{ZoneRef: CloudStackResourceRef{Type: "name", Value: "zone2"}, Network: CloudStackResourceRef{Type: "name", Value: "GuestNet2"}},
```

Comment:
I feel like the default should only be one zone. Most customers would only use one, right?

Reply:
I agree that single zone should be the common case, the point here is to let customer know multiple zone is supported. I also recognize that the array grammar in yaml indicates it support multiple zones.

Reply:
Keep it simple, multiple zones is probably unnecessary. Its just more for the end-user to have to delete/think about.